EziamakaNV/epicmail
# #164612579 Implement api create and own a group

### What does this PR do?
This endpoint allows a user to create a and own a group

### Description of Task to be completed?
Create the routes to handle requests concerning the groups
Ensure that the name and userid are present in the request parameters
Insert the new group into the groups table
Respond with the appropriate response codes (201 for create)

### How should this be manually tested?
Clone the repository, CD into API directory and type "npm start" on the command line (starts the server)
Using Postman, make a POST request to "localhost:8080/api/v2/groups"
In the body Request make sure the name parameter is entered

#### What are the relevant pivotal tracker stories?
- PT Story link - ([#164612579](https://www.pivotaltracker.com/story/show/164612579))
